backend/infrahub/core/node/resource_manager/ip_prefix_pool.py

```diff
@@ -3,14 +3,16 @@
 import ipaddress
 from typing import TYPE_CHECKING, Any, Optional
 
+from netaddr import IPSet
+
 from infrahub.core import registry
 from infrahub.core.ipam.reconciler import IpamReconciler
 from infrahub.core.query.ipam import get_subnets
 from infrahub.core.query.resource_manager import (
     PrefixPoolGetReserved,
     PrefixPoolSetReserved,
 )
-from infrahub.pools.prefix import PrefixPool
+from infrahub.pools.prefix import get_next_available_prefix, get_prefix_type
```

Comment:
I'm not sure if this is the best place for these helper functions -- happy to move them to wherever makes sense.